interface/client/appStart.js

```diff
@@ -35,17 +29,20 @@ mistInit = function(){
             });
         }
 
-
         Tabs.upsert({_id: 'wallet'}, {
             url: 'https://wallet.ethereum.org',
             position: 1,
             permissions: {
                 admin: true
             }
-        });        
-    })
-    .then(function() {
-        window.trigger('mist-ready');
```

Comment:
It was for future usage. A dapp would be able to listen to the `mist-ready` event to know when all data was loaded and ready. I'd prefer if this was re-instated.